packages/core/src/BillingDataRow.ts

```diff
@@ -13,13 +15,14 @@ export default abstract class BillingDataRow {
   public region: string
   public cloudProvider: string
   public timestamp: Date
-  public vCpus: number
+  public vCpus?: number
   public vCpuHours: number
   public gpuHours: number
   public machineType: string
   public seriesName: string
   public instanceType: string
   public replicationFactor: number
+  public tags: TagCollection
```

Comment:
i think this needs to be marked as optional

Reply:
Resolved in https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/pull/904/commits/4b2923d0de73dff9e9473a56b819709069a02ec3.